### PR TITLE
Test tuning cni whitelist update

### PR DESCRIPTION
**Pull request by mmirecki**
Add test for sysctl whitelist update.

NOTE: This PR requires the following PR to be merged first:  https://github.com/openshift/cluster-network-operator/pull/1573

### CODE REVIEW

**Comment by mmirecki** [comment]
@knobunc @deads2k @dcbw @danwinship @smarterclayton @bparees @mfojtik 
Can you please take a look at this?

File: test/extended/networking/tuning.go [hunk 1]
```diff
@@ -253,3 +303,13 @@ func createTuningNADWithBridgeName(config *rest.Config, namespace, nadName, brid
 	nadConfig := fmt.Sprintf(`{"cniVersion":"0.4.0","name":"%s","plugins":[{"type":"bridge","bridge":"%s","ipam":{"type":"static","addresses":[{"address":"10.10.0.1/24"}]}}%s]}`, nadName, bridgeName, sysctlString)
 	return createNetworkAttachmentDefinition(config, namespace, nadName, nadConfig)
 }
+
+func updateALlowlistConfig(sysctls string, client clientset.Interface) error {
```
Comment (bparees) [inline test/extended/networking/tuning.go:307]:
```suggestion
func updateAllowlistConfig(sysctls string, client clientset.Interface) error {
```

**Comment by mmirecki** [comment]
@bparees Could you please have another look?
Thanks

**Comment by bparees** [comment]
> @bparees Could you please have another look?

this needs to be reviewed/approved by the networking team, @trozet @knobunc 

also @knobunc is this owners file up to date?
https://github.com/openshift/origin/blob/master/test/extended/networking/OWNERS

**Comment by mmirecki** [comment]
@trozet @knobunc Can you take a look? PLEASE! This is waiting for a review for way over a month now!

**Comment by mmirecki** [comment]
@knobunc @deads2k @dcbw @danwinship @smarterclayton  @mfojtik
Can you please take a look at this?

**Pull request merged**